### PR TITLE
Add initial Cosmos module

### PR DESCRIPTION
**Pull request by alpe**
Setup `custom` contract message handler and SDK module

### CODE REVIEW

File: demo/app/app.go [hunk 1]
```diff
@@ -579,6 +585,15 @@ func NewMeshApp(
 		panic(fmt.Sprintf("error while reading wasm config: %s", err))
 	}
 
+	meshMessageHandler := wasmkeeper.WithMessageHandlerDecorator(func(nested wasmkeeper.Messenger) wasmkeeper.Messenger {
+		return wasmkeeper.NewMessageHandlerChain(
+			nested,
+			// append our custom message handler
+			meshseckeeper.NewDefaultCustomMsgHandler(app.MeshSecKeeper),
```
Comment (alpe) [inline demo/app/app.go:592]: Setup our custom message handler for CosmWasm

File: docs/proto/proto-docs.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
+<!-- This file is auto-generated. Please do not modify it yourself. -->
```
Comment (alpe) [inline docs/proto/proto-docs.md:1]: copied from wasmd

File: x/meshsecurity/contract/in_message.go [hunk 3]
```diff
@@ -0,0 +1,30 @@
+package contract
```
Comment (alpe) [inline x/meshsecurity/contract/in_message.go:1]: copied from spike1

**Comment by ethanfrey** [comment]
You split it and still 4000 lines? You are a machine.
I mean around 1600 look autogenerated protobuf stuff, but still that is a lot of coding....

**Comment by alpe** [comment]
>Let's keep the scope tight here (adding all the wiring, etc)...
👌 I will move more code into a new PR. Please use squash+merge with this PR

File: x/meshsecurity/keeper/handler_plugin_test.go [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+	"github.com/cometbft/cometbft/libs/rand"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/address"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	wasmvmtypes "github.com/CosmWasm/wasmvm/types"
+)
+
+func TestCustomMeshSecDispatchMsg(t *testing.T) {
```
Comment (pinosu) [inline x/meshsecurity/keeper/handler_plugin_test.go:20]: Good tests 👍

**Review by pinosu — APPROVED** [review]
LGTM ! 👍

**Pull request merged**